test/manual_test_sample.py

```diff
@@ -0,0 +1,60 @@
+from sample_metadata.apis import SampleApi, AnalysisApi
+from sample_metadata.models import NewSample, AnalysisModel
+
+
+PROJ = 'test_project'
+
+
+def run_test():
+    """Run test"""
+    sapi = SampleApi()
+    aapi = AnalysisApi()
+
+    def _print_samples():
+        samples = sapi.get_samples(
+            body_get_samples_by_criteria_api_v1_sample_post={
+                'project_ids': [PROJ],
+                'active': True,
+            }
+        )
```

Comment:
is this making a real connection to the API?

Reply:
This is, it's an older test I've renamed / pulled from Vlad's older PR - I'd like to rewrite this test, or come up with a new testing class that allows you to use the SM API, because for that it will have to:
- spin up a web server
- Probably ask the SM API to _NOT_ generate you an auth token because it relies on some auth'd user
- Ask the SM API to hit this local server instead, which might not necessarily run on port 8000 (the default `local` port).

Reply:
When I was unit-testing a method that called the SM API I tried to spoof the responses using requests-mock, but struggled because the client required authentication in a way that I couldn't easily see how to mock. If you end up with a way to mock SMAPI/DB responses please let me know!